src/main/java/com/thebois/models/world/structures/AbstractStructure.java

```diff
@@ -11,27 +15,39 @@
 
     private Position position;
     private StructureType structureType;
+    private IInventory allNeededItems;
```

Comment:
Think we talked about this being a hardcoded list of for each building? Then we would compare it that way, opinions?